blog/views.py

```diff
@@ -1,32 +1,338 @@
 
 
 from django.shortcuts import render, get_object_or_404
-from .models import Post, Comment, PostViewsCount
-from .cmtforms import CommentForm
+from django.urls.base import reverse
+from django.urls import reverse, reverse_lazy
+from django.template.defaultfilters import slugify
+
 from django.views.generic import ListView
 from django.http import HttpResponseRedirect
 from intern.models import ReviewRating
-class PostListView(ListView):
+from django.db.models import Count,Exists   
+from django.core.exceptions import ObjectDoesNotExist
+
+from .models import *
+from .cmtforms import CommentForm
+
+def show_blog(request):
+    #show top views (posts)
+    for obj in Post.objects.filter(slug= None):
+        
+        print(obj)
```

Comment:
I would suggest using Django logging to log info https://docs.djangoproject.com/en/3.2/topics/logging/

Reply:
Sorry about it. I forget delete it. I want to update for posts which are created before adding "slug" field in model post